config/helm/deployment-args-kubeadm-bootstrap-controller-manager.yaml

```diff
@@ -13,4 +13,3 @@ spec:
             - --feature-gates=MachinePool=true,KubeadmBootstrapFormatIgnition=true
             - --watch-filter={{ .Values.watchFilter }}
             - --v=0
-          image: '{{.Values.images.domain}}/{{.Values.images.bootstrap.name}}:{{.Values.images.bootstrap.tag | default .Values.images.tag}}'
```

Comment:
we don't need this line as there is already an `image` section in `kustomize` which we can use for the `image: ` value replacement (see `kustomize.yaml` changes in this PR)